MBS-7669 Add filter troubleshooting docs (#378)
avito-tech/avito-android: docs/content/docs/test/Runner.md

./gradlew instrumentationLocal -PincludedAnnotation="package.AnotherAnnotationClassName" 
```

> ## Find out how filters applied

Reviewer: ```suggestion
## Find out how filters were applied
```

> If build finished successfully there will be files with debug information

Author: ```suggestion
If build finishes successfully It will produce files with debug information
```

> If build finished successfully It will produce files in which you must find filters debug information

Reviewer: ```suggestion
If build finished successfully there will be files with debug information
```
I think we have no reason for "must".

Files will be located at: \
 `<Project root folder>/outputs/<subproject name>/instrumentation/<instrumentation task name>/filter`

### Find out what filter config was

> Look at file `filter-config.json`

Reviewer: Can we add some details about the content? Maybe a small example.

Author: It's too simple. If we add some detail we must support those details. I don't want to support those files behavior

Reviewer: Ok

### Find out what filters applied

Look at file `filters-applied.json`

### Find out what tests were filtered

> 0. Look at file `filters-excludes.json`

Reviewer: Why do we need an ordered list? It contains only one element.

> - You may find filtered tests by the filter by the filter name declared in `filters-applied.json`

Reviewer: It's hard for me to understand the reference. Maybe we can simplify it.
